include/cantera/clib/ctonedim.h

```diff
@@ -59,14 +59,14 @@ extern "C" {
 
     CANTERA_CAPI int inlet_setSpreadRate(int i, double v);
 
-    CANTERA_CAPI int stflow_new(int iph, int ikin, int itr, int itype);
-    CANTERA_CAPI int stflow_setTransport(int i, int itr);
-    CANTERA_CAPI int stflow_enableSoret(int i, int iSoret);
-    CANTERA_CAPI int stflow_setPressure(int i, double p);
-    CANTERA_CAPI double stflow_pressure(int i);
-    CANTERA_CAPI int stflow_setFixedTempProfile(int i, size_t n, const double* pos,
+    CANTERA_CAPI int flow1D_new(int iph, int ikin, int itr, int itype);
+    CANTERA_CAPI int flow1D_setTransport(int i, int itr);
+    CANTERA_CAPI int flow1D_enableSoret(int i, int iSoret);
+    CANTERA_CAPI int flow1D_setPressure(int i, double p);
+    CANTERA_CAPI double flow1D_pressure(int i);
+    CANTERA_CAPI int flow1D_setFixedTempProfile(int i, size_t n, const double* pos,
             size_t m, const double* temp);
-    CANTERA_CAPI int stflow_solveEnergyEqn(int i, int flag);
+    CANTERA_CAPI int flow1D_solveEnergyEqn(int i, int flag);
 
```

Comment:
This is foregoing any attempt to maintain stable API for `clib`, then? I'm fine with that, but just wanted to make sure that was the intention.

Reply:
The main place where this is relevant is the experimental MATLAB toolbox, so this may be a feature.

Reply:
I ended up triggering errors for the old `clib` functions with the `stflow_` prefix. That way the symbols are still defined, which should make the shared library a little more stable. I'm not sure that any interfaces other than MATLAB use clib's 1D features.